spaces/S000119/properties/P000055.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000119
+property: P000055
+value: true
+---
+
+Similar to {S116|P055}.
```

Comment:
```suggestion
Viewing $X$ as a subspace of $Y=$ {S176}, we have

$\quad\quad X=\overline{X} \cap \bigcap_{n \geq 2}\mathbb{R}^2
\setminus([\frac{1}{n},\infty)\times[1,\infty))$

with $\overline X$ a $G_\delta$ set in $Y$
since {S176|P132}.
Therefore $X$ is a $G_\delta$ set in $Y$ and the assertion follows
since {S176|P55}.
```
we may as well be explicit